packages/fxa-auth-server/lib/sentry.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Hoek = require('@hapi/hoek');
+const Sentry = require('@sentry/node');
+
+const getVersion = require('./version').getVersion;
+
+// Matches uid, session, oauth and other common tokens which we would
+// prefer not to include in Sentry reports.
+const TOKENREGEX = /[a-fA-F0-9]{32,}/gi;
+const FILTERED = '[Filtered]';
+
+/**
+ * Filter a sentry event for PII in addition to the default filters.
+ *
+ * Current replacements:
+ *   - A 32-char hex string that typically is a FxA user-id.
+ *
+ * Data Removed:
+ *   - Request body.
+ *
+ * @param {Sentry.Event} event
+ */
+function filterSentryEvent(event) {
+  if (event.breadcrumbs) {
```

Comment:
This didn't filter breadcrumbs for me. Looks like the object is structured like this

<img width="925" alt="Screen Shot 2019-10-25 at 12 18 08 PM" src="https://user-images.githubusercontent.com/1295288/67587315-99555a80-f721-11e9-998c-79a92c813176.png">

Ref: https://sentry.prod.mozaws.net/operations/fxa-ben-test/issues/6539388/

Reply:
I pushed another change that hopefully will grab these, not sure why I couldn't iterate through the event to re-assign them.

Reply:
Hmm doesn't look like the breadcrumbs are being [filtered](https://sentry.prod.mozaws.net/operations/fxa-ben-test/issues/6539808/?query=is:unresolved). When testing locally, I couldn't trigger a breakpoint in `beforeBreadcrumb` but could in `filterSentryEvent`. Is there another config that would need to be enabled?

Reply:
Ok, *now* it should be filtering them. I configured it incorrectly it seems.